File: src/main/java/top/dsbbs2/bukkitcord/utils/command/IChildCommand.java
package top.dsbbs2.bukkitcord.utils.command;

import top.dsbbs2.bukkitcord.api.ICommand;
import top.dsbbs2.bukkitcord.api.ICommandSender;

public interface IChildCommand {
    default void onCommand(ICommandSender sender, ICommand cmd,String[] args) {}
    default String getPermission(){return "";}
    default String getUsage(){return "";}
}

File: src/main/java/top/dsbbs2/bukkitcord/utils/command/IChildCommand.kt
package top.dsbbs2.bukkitcord.utils.command

import top.dsbbs2.bukkitcord.api.ICommand
import top.dsbbs2.bukkitcord.api.ICommandSender

abstract class IChildCommand {
    fun onCommand(sender: ICommandSender?, cmd: ICommand?, args: Array<String?>?) {}
    val permission: String
        get() = ""

    val usage: String
        get() = ""
}